avocado/core/result.py

```diff
@@ -109,3 +109,11 @@ def check_test(self, state):
         else:
             self.errors += 1
         self.end_test(state)
+
+    @property
+    def rate(self):
+        total = float(self.tests_total - self.skipped - self.cancelled)
+        if not total:
+            return 0
```

Comment:
This is a really small nitpick, but, this makes the return either a int or a float.  The UI should take care of the differences, but, its probably be a good idea to always return a float.